pkg/operator/controller/ingress/controller_test.go

```diff
@@ -337,6 +353,18 @@ func TestSetDefaultPublishingStrategyHandlesUpdates(t *testing.T) {
 			expectedResult: false,
 			expectedIC:     makeIC(spec(elb()), status(nlb())),
 		},
+		{
+			name:           "loadbalancer ELB connection idle timeout changed from unset to 2m",
+			ic:             makeIC(spec(elbWithIdleTimeout(&metav1.Duration{Duration: 2 * time.Minute})), status(elb())),
+			expectedResult: true,
+			expectedIC:     makeIC(spec(elbWithIdleTimeout(&metav1.Duration{Duration: 2 * time.Minute})), status(elbWithIdleTimeout(&metav1.Duration{Duration: 2 * time.Minute}))),
+		},
+		{
+			name:           "loadbalancer ELB connection idle timeout changed from 2m to unset",
+			ic:             makeIC(spec(elb()), status(elbWithIdleTimeout(&metav1.Duration{Duration: 2 * time.Minute}))),
+			expectedResult: true,
+			expectedIC:     makeIC(spec(elb()), status(elbWithIdleTimeout(nil))),
+		},
```

Comment:
Maybe add a test for a bad value to see what happens.

Reply:
Is the "loadbalancer ELB connection idle timeout changed from unset to -1s" test what you had in mind?